spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/container-factory.adoc

```diff
@@ -38,3 +38,18 @@ public KafkaListenerContainerFactory<?> kafkaListenerContainerFactory() {
 }
 ----
 
+Starting with version 3.1.0, it's also possible to apply the same kind of customization on a single listener by specifying the bean name of a
```

Comment:
```suggestion
Starting with version 3.1, it's also possible to apply the same kind of customization on a single listener by specifying the bean name of a
```
We only use `x.y.z` when `z` is greater than zero.

Also add a paragraph to `../whats-new.adoc`.